### DISCUSSION TITLE
[@types/jest] missing property 'toStrictEqual' does not exist

### DISCUSSION THREAD

**Discussion by gyula-s** [DefinitelyTyped/DefinitelyTyped#56115] [Issues with a @types package] [answered]
One of the major things, that prevents me to use .ts for my unit tests, is because the following error:
`Property 'toStrictEqual' does not exist on type 'Matchers<void>'.ts(2339)`

It looks that the `toStrictEqual` is missing from the Matchers interface.

I would like to open a PR to address this, where I'd add the missing entry to the `interface Matchers<R>` in the `@types/jest`

[a comment by an automated account is not shown]

**Comment by tonyhallett** [comment, marked as answer]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6ec65f01639eac32c493ca95b9dd2705801a7ef4/types/jest/index.d.ts#L1015

**Reply by gyula-s** [reply]
I'm so sorry. I knew there was something I wanted to do before I raised this... Check if I'm using an older version. Sorry for wasting your time.
